тетраэдр/тетраэдр/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace тетраэдр
+{
+    class Program
+    {
+        static void Main()
+        {
+            double x;
+            double b;
+            double c;
+           Console.WriteLine("Введите длину ребра тетраэдра:");
+            x = double.Parse(Console.ReadLine());
+           
+            b = x*x*x*Math.Sqrt(2)/12;
+            c = x * x * Math.Sqrt(3);

```

Comment:
И разделяйте знаки операций пробелами. Так код легче читается.
```suggestion
            double b = x * x * x * Math.Sqrt(2) / 12;
            double c = x * x * Math.Sqrt(3);

```